### PR TITLE
Add newline for REPL output

### PR DESCRIPTION
**Pull request by gregli-msft**
The current REPL library doesn't provide whitespace between the expression and the output.  With the REPL here, we have the ability to color the prompt which helps.  to visually group the expression with the output.  But with the PAC CLI, we don't have this ability and it all bleeds together.  This is particularly true for Power Fx where an expression could cause recalc of other values which are rightly shown in the output.

In short, we want to change this:
image

Into the slightly longer, but much easier to parse and read:
image

We could have fixed this just in the PAC CLI, however I prefer this whitespace here too and consistency is good.  This also ensures that table formatting works correctly with added newlines.  It also mirrors the tests cases.

### CODE REVIEW

File: src/libraries/Microsoft.PowerFx.Repl/Services/StandardFormatter.cs [hunk 1]
```diff
@@ -28,6 +28,10 @@ public class StandardFormatter : ValueFormatter
         // Set to Int32.MaxInt to remove restriction.
         public int MaxTableRows { get; set; } = 10;
 
+        // Append a newline to the end of a formatted table.
+        // If the REPL does not include a newline for the prompt, set this to true for proper spacing.
+        public bool FormatTableNewLine { get; set; } = false;
```
Comment (MikeStall) [inline src/libraries/Microsoft.PowerFx.Repl/Services/StandardFormatter.cs:33]: Does anybody ever set this to true?
It would be nice if we didn't need a separate flag here - we're basically double capturing this (in Repl.Prompt, and here).

**Review by MikeStall — APPROVED** [review]

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Repl/Services/StandardFormatter.cs (hunk 1 above)
Comment (gregli-msft) [reply src/libraries/Microsoft.PowerFx.Repl/Services/StandardFormatter.cs:33]: No, nobody sets the FormulatTableNewLine flag today.  If Prompt didn't include "\n", then it should be set to true.  In theory the formatter could go look at the REPL Prompt property but that seemed like more dependency than this was worth and currently the formatters are a leaf node and unaware of the hosting REPL which is a good thing.
